evm/src/arithmetic/mod.rs

```diff
@@ -137,6 +150,80 @@ impl Operation {
     }
 }
 
+pub trait Traceable<F: PrimeField64> {
```

Comment:
nit: Do we expect this trait to be implemented on something else than `Operation`?

Reply:
No, good point. I'll just put the function in the `impl Operation` block.